src/Compilers/CSharp/Test/Emit2/Emit/NumericIntPtrTests.cs

```diff
@@ -11484,6 +11484,75 @@ .maxstack  4
 ");
         }
 
+        [WorkItem(63348, "https://github.com/dotnet/roslyn/issues/63348")]
+        [Fact]
+        public void ConditionalStackalloc()
+        {
+            var source =
+@"using System;
+class Program
+{
+    static int F(int n)
+    {
+        Span<char> s = n < 10 ? stackalloc char[n] : new char[n];
```

Comment:
Where is the `IntPtr` instance coming from here? Is it the common type of the ternary?

Reply:
The `stackalloc` size is converted from `UInt32` to an unsigned native integer in lowering - see [LocalRewriter_StackAlloc.cs#L139](https://github.com/dotnet/roslyn/blob/main/src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_StackAlloc.cs#L139).